README.md

```diff
@@ -139,6 +139,61 @@ ros2 launch mini_pupper_bringup bringup.launch.py
 ros2 run teleop_twist_keyboard teleop_twist_keyboard
 # Then control Mini Pupper with the keyboard
 ```
+### 2.2.2 Test Mapping
+  - Bring up real mini pupper
+ ```bash
+ # Terminal 1 (ssh to real mini pupper)
+. ~/ros2_ws/install/setup.bash
+ros2 launch mini_pupper_bringup bringup.launch.py  # on real mini pupper
+ ```
+  - Mapping on PC
+ ```bash
+ # Terminal 2 (on PC)
+ . ~/ros2_ws/install/setup.bash
+ ros2 launch mini_pupper_navigation slam.launch.py  # on PC
+ ```
+ - Keyboard control  
+Use the keyboard to remotely control the mini pupper to complete the mapping.
+```bash
+# Terminal 3 (on PC)
+ros2 run teleop_twist_keyboard teleop_twist_keyboard
+```
+- Save the map  
+If you want to save the map:
```

Comment:
```suggestion
```

I think it is necessary to save the map for navigation, so how about save the map in general?